CONFIGURE.md

```diff
@@ -0,0 +1,65 @@
+## Configuring this docker image
+
+It's possible to config both the build process to create this docker image as well as how the TrueBlocks executable (called `chifra`) works internally. Both of these possibilities is described below.
```

Comment:
`Both of these possibilities is described below` -> `Both of these possibilities **are** described below`?

Reply:
You have no idea how much I appreciate this comment (being an X teacher of writing).